v03_pipeline/api/app.py

```diff
@@ -1,17 +1,80 @@
-from aiohttp import web
+import json
+import os
+import traceback
 
-from v03_pipeline.lib.tasks import *  # noqa: F403
+import aiofiles
+import aiofiles.os
+from aiohttp import web, web_exceptions
+
+from v03_pipeline.api.model import LoadingPipelineRequest
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.paths import loading_pipeline_queue_path
+
+logger = get_logger(__name__)
+
+
+@web.middleware
+async def error_middleware(request, handler):
+    try:
+        return await handler(request)
+    except web.HTTPError:
+        logger.exception('HTTPError')
+        raise
+    except Exception as e:
+        logger.exception('Unhandled Exception')
+        error_reason = f'{e}: {traceback.format_exc()}'
+        raise web.HTTPInternalServerError(reason=error_reason) from e
+
+
+async def loading_pipeline_enqueue(request: web.Request) -> web.Response:
```

Comment:
The idea here is for this process to accept requests and manage a queue of length one (just a file on the filesystem).  It did not seem feasible for this process itself to run the luigi tasks, since they are likely to take many hours.